mne/io/egi/tests/test_egi.py

```diff
@@ -157,4 +157,39 @@ def test_io_egi_pns_mff():
         assert_array_equal(mat_data, raw_data)
 
 
+@requires_testing_data
+def test_io_egi_pns_mff_bug():
+    """Test importing EGI MFF with PNS data (BUG)"""
+    egi_fname_mff = op.join(data_path(), 'EGI', 'test_egi_pns_bug.mff')
+    raw = read_raw_egi(egi_fname_mff, include=None, preload=True,
+                       verbose='error')
```

Comment:
why `error` here? If it's for a warning about samples missing, it might be better to ensure the expected warning appears, e.g.:
```
with warnings.catch_warnings(record=True) as w:
    ...
assert any('EGI PSG sample bug' in str(ww.message) for ww in w)
```